usr/share/rear/backup/BORG/default/500_make_backup.sh

```diff
@@ -3,35 +3,74 @@
-    include_list+=("$i ")
-done
+while IFS= read -r include; do
+    include_list+=( "$include" )
+done < "$TMP_DIR/backup-include.txt"
 
 # User might specify some additional output options in Borg.
 # Output shown by Borg is not controlled by `rear --verbose' nor `rear --debug'
-local borg_additional_options=''
+# only, if BORGBACKUP_SHOW_PROGRESS is true.
 
-is_true $BORGBACKUP_SHOW_PROGRESS && borg_additional_options+='--progress '
-is_true $BORGBACKUP_SHOW_STATS && borg_additional_options+='--stats '
+# shellcheck disable=SC2168
+local borg_additional_options=()
 
-# Start actual Borg backup.
-Log "Creating archive ${BORGBACKUP_ARCHIVE_PREFIX}_$BORGBACKUP_SUFFIX \
-in repository $BORGBACKUP_REPO"
+BORGBACKUP_CREATE_SHOW_PROGRESS=${BORGBACKUP_CREATE_SHOW_PROGRESS:-$BORGBACKUP_SHOW_PROGRESS}
+BORGBACKUP_CREATE_SHOW_STATS=${BORGBACKUP_CREATE_SHOW_STATS:-$BORGBACKUP_SHOW_STATS}
+BORGBACKUP_CREATE_SHOW_LIST=${BORGBACKUP_CREATE_SHOW_LIST:-$BORGBACKUP_SHOW_LIST}
+BORGBACKUP_CREATE_SHOW_RC=${BORGBACKUP_CREATE_SHOW_RC:-$BORGBACKUP_SHOW_RC}
+
+is_true "$BORGBACKUP_CREATE_SHOW_PROGRESS" && borg_additional_options+=( --progress )
+is_true "$BORGBACKUP_CREATE_SHOW_STATS" && borg_additional_options+=( --stats )
+is_true "$BORGBACKUP_CREATE_SHOW_LIST" && borg_additional_options+=( --list --filter AME )
+is_true "$BORGBACKUP_CREATE_SHOW_RC" && borg_additional_options+=( --show-rc )
+is_true "$BORGBACKUP_EXCLUDE_CACHES" && borg_additional_options+=( --exclude-caches )
+is_true "$BORGBACKUP_EXCLUDE_IF_NOBACKUP" && borg_additional_options+=( --exclude-if-present .nobackup )
+[[ -n $BORGBACKUP_TIMESTAMP ]] && borg_additional_options+=( --timestamp "$BORGBACKUP_TIMESTAMP" )
+
+# Borg writes all log output to stderr by default.
+# See https://borgbackup.readthedocs.io/en/stable/usage/general.html#logging
+#
+# If we want to have log output from Borg appearing in rear logs, we don't have
+# to do anything, since Borg logs to stderr and that is what rear is saving in
+# Logfile.
+#
+# If `--progress` is used for `borg create` we don't want the output in rear
+# log file, since it contains control sequences. If not used, we want Borg
+# output in rear log file, the amount of logs written by Borg is determined by
+# other options above e.g. by `--stats` or `--list --filter=AME`.
 
-borg create --one-file-system $borg_additional_options $verbose \
-$BORGBACKUP_OPT_COMPRESSION $BORGBACKUP_OPT_REMOTE_PATH \
-$BORGBACKUP_OPT_UMASK --exclude-from $TMP_DIR/backup-exclude.txt \
-${borg_dst_dev}${BORGBACKUP_REPO}::\
-${BORGBACKUP_ARCHIVE_PREFIX}_$BORGBACKUP_SUFFIX \
-${include_list[@]} 0<&6 1>&7 2>&8
+# https://github.com/rear/rear/pull/2382#issuecomment-621707505
+# Depending on BORGBACKUP_SHOW_PROGRESS and VERBOSE variables
+# 3 cases are there for `borg_create` to log to rear log file or not.
+#
+# 1. BORGBACKUP_SHOW_PROGRESS true:
+#    No logging to rear log file because of control characters.
+#
+# 2. VERBOSE true:
+#    stderr (2) is copied to real stderr (8):
```

Comment:
The comment does no longer match what the code does because
now both stdout and stderr are copied to the ReaR log file
and redirected to the user's terminal.

Reply:
See a5948d5578bd5ab9155a4650259b8eed34e79c74.